Source/x11/XGServerEvent.m

```diff
@@ -484,7 +484,8 @@ - (void) processEvent: (XEvent *) event
                 XSendEvent(dpy, cWin->parent, True,
                            ButtonPressMask, &xEvent);
                 XFlush(dpy);
-                break;
+                if (clickCount != 2)
```

Comment:
Why don't you check for "clickCount == 1", which would be closer to the old behaviour? This isn#T important but changes the result of a triple click.

Reply:
There was some glitches with old -back and -gui for double-clicks. I've just checked it with current code - everything seems correct.